contentcuration/contentcuration/frontend/channelEdit/components/edit/EditModal.vue

```diff
@@ -49,7 +49,7 @@
               clipped
               app
               style="height: calc(100% - 64px);"
-              :minWidth="150"
+              :minWidth="250"
```

Comment:
This only achieves the desired outcome because the [default width is the `minWidth`](https://github.com/learningequality/studio/blob/unstable/contentcuration/contentcuration/frontend/shared/views/ResizableNavigationDrawer.vue#L71). May still want users to be able to resize the panel to less than 250px, so I think a slightly better fix would be to add a default width prop to the `ResizableNavigationDrawer` which if unset, maintains the current behavior.

Reply:
Let me understand , What I have to do is , First I will make a prop naming it as defaultWidth so that user can change it by him/her self otherwise minWidth = 250px will be there if they doesn't change it.
Anything else I am missing in this , please tell.

Reply:
Yes, I think you've understood it correctly. One clarification though, the `ResizableNavigationDrawer` component is resizable by the _user_, so it allows the user to resize it between the widths of `minWidth` and `maxWidth`. The component also remembers what size the user preferred, but if the user hadn't resized it, then the component defaults to `minWidth` which is too small by default.

So yes, the change I suggested, for adding a `defaultWidth` prop to `ResizableNavigationDrawer`, would allow _developers_ like yourself to configure a sensible default, considering `150` is too small for a default width here. After you implement the `defaultWidth` prop in `ResizableNavigationDrawer`, you can then utilize it here to set the default width to `250`.

Reply:
To help make it easier, once you add the `defaultWidth` prop to `ResizableNavigationDrawer`, you may update [this line](https://github.com/learningequality/studio/blob/unstable/contentcuration/contentcuration/frontend/shared/views/ResizableNavigationDrawer.vue#L71) to this:
```
width: parseFloat(localStorage[localStorageName]) || this.defaultWidth || this.minWidth,
```
which returns the first non-falsy value from left to right, where the first `parseFloat` bit is where it retrieves the user's previous size from browser storage.

Reply:
Ok thanks, I'll let you know about this.
Actually yesterday I got fever so please give me 2-3 days. After that I'll do this.